circle.yml

```diff
@@ -0,0 +1,19 @@
+checkout:
+  post:
+    - git fetch --unshallow || true
+    - git fetch --tags
+
+dependencies:
+  override:
+    - pip install tox coverage
+    - pyenv local 2.7.10 2.6.8
+
+test:
+  pre:
+    - coverage erase
+  override:
+    - tox -v --recreate -- --with-xunit
```

Comment:
Do we always want to recreate? CircleCI has a fairly useful caching feature, which could speed up builds quite a bit.